evm_loader/cli/src/main.rs

```diff
@@ -1193,6 +1196,77 @@ fn command_neon_elf(
     });
 }
 
+fn command_neon_elf(
+    config: &Config,
+    program_location: Option<&str>,
+) -> CommandResult {
+    if let Some(program_location) = program_location {
+        // Reading program data from file
```

Comment:
This two comments means it can be wrapped with two methods. `read_program_data_from_file` and `read_program_data_from_account`.
This is recommendation but not requirement

Reply:
Fixed